### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by veni-vidi-code**
First Codeowners

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by VincentRPS** [comment]
i forgor 💀 what is codeowners for?

**Review by VincentRPS — APPROVED** [review]
When the imposter is a sussy baka

**Review by Questia — COMMENTED** [review]
by the way i just changed my github name and basically everyother name so you might wanna change that

**Review by VincentRPS — COMMENTED** [review]
why this hasn't been merged yet?

File: .github/CODEOWNERS
Comment (TAG-Epic) [inline .github/CODEOWNERS:1]: Not spesific enough, should specify how and where.

File: .github/CODEOWNERS
Comment (TAG-Epic) [inline .github/CODEOWNERS:10]:
```suggestion
# Voice Recording
```

Unmerged, should not be added yet.

**Review by TAG-Epic — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS
Comment (VincentRPS) [inline .github/CODEOWNERS:18]: same with what epic said this shouldn't be added(or at least commented) until it is merged

Comment (VincentRPS) [inline .github/CODEOWNERS:18]: Specify Better
```suggestion
# Application And Interaction Commands
```

**Comment by RPSMain** [comment]
@TAG-Epic all the changes you wanted should be done now

**Pull request merged**
